README.md

```diff
@@ -169,3 +170,6 @@ To find out more about ADRs have a read of this article: http://thinkrelevance.c
 ## Links
 
 * [CI Pipeline](https://circleci.com/gh/redbadger/workflows/pride-london-app): View and debug builds
+
+We would like to thank [BrowserStack](https://www.browserstack.com) for allowing us to use their resources to test the Pride in London App for free! 
```

Comment:
Just to keep the consistency, make it a list item: `* We would like to thank..` 👍